src/main/java/me/crespel/karaplan/web/api/v1/PlaylistController.java

```diff
@@ -91,6 +92,13 @@ public Playlist savePlaylist(@PathVariable Long playlistId, @RequestBody Playlis
 				.setReadOnly(playlist.getReadOnly());
 		return playlistService.save(playlistToSave, user);
 	}
+	
+	@PostMapping("/{playlistId}/sort/custom")
+	@ApiOperation("Change song order")
+	public Playlist changeOrder(@PathVariable Long playlistId, @RequestBody List<Long> songIdList, @ApiIgnore @AuthenticationPrincipal(expression = "user") User user) {

```

Comment:
Consider renaming this method to `sortPlaylistCustom()` and put it just after `sortPlaylist()` since their path look alike.